buildSrc/subprojects/license-texts/src/main/kotlin/com/github/vlsi/gradle/license/EnumGeneratorTask.kt

```diff
@@ -159,6 +169,18 @@ open class EnumGeneratorTask @Inject constructor(objectFactory: ObjectFactory) :
                             .addStatement("return idToInstance[id]")
                             .build()
                     )
+                    .addFunction(
+                        FunSpec.builder("fromUri")
+                            .addParameter("uri", String::class)
```

Comment:
This should probably be `URI:class` for consistency. I'm not sure yet.

Reply:
Can't really judge what implications that would have